build/images/ovs/apply-patches.sh

```diff
@@ -80,8 +80,16 @@ if version_get "$OVS_VERSION" "2.13.0" && version_lt "$OVS_VERSION" "2.14.0" ; t
         git apply
 fi
 
+# Starting from version 5.7.0, strongSwan no more supports to specify a configuration parameter with
+# the path delimited by dots in a configuration file. This patch fixes the strongSwan configuration
+# parameters that ovs-monitor-ipsec writes, to comply with the new strongSwan format.
+# After the patch is merged into OVS upstream, we should remove this workaround to apply the patch
+# here.
```

Comment:
The first step will be to change the URL for the patch actually, so:
> After the patch is merged into OVS upstream, we should change the URL to an upstream one.

We won't be able to remove this altogether until there is a new OVS release with the patch.

Reply:
Revised the comments. Check.